optaweb-employee-rostering-backend/Dockerfile

```diff
@@ -1,5 +1,5 @@
-FROM adoptopenjdk/openjdk8:ubi-minimal-jre
-COPY target/*-exec.jar /opt/app/optaweb-employee-rostering.jar
+FROM adoptopenjdk/maven-openjdk11:latest
```

Comment:
Why? UBI for 11 exists just as well:
https://hub.docker.com/layers/adoptopenjdk/openjdk11/ubi-minimal-jre/images/sha256-02e631ef7445a5c84a021f494fc5c831a6b4264307570f0fa67e0fd681beed71?context=explore

Reply:
I copied the one from standalone; it can probabally work for UBI